js/app.js

```diff
@@ -0,0 +1,745 @@
+console.log('HOMEWORK');
+
+
+//1. VARIABLES AND DATATYPES
+
+// How do we assign a value to a variable?
+
+//a. We assign a value to a variable with '='
+
+
+// How do we change the value of a variable?
+
+//a. By using the name of that same variable = to a different value.
+
+// How do we assign an existing variable to a new variable?
+
+// a. By declaring a new variable and making it equal to the name of the existing variable.
+
+// Remind me, what are declare, assign, and define?
+
+//a. when you declare a variable reserves the space for a variable to be used later.
+//b. when you are assigning a variable you are setting the value.
+//c. when you define a variable comprises both the declaration and assignment.
+
+// What is pseudocoding and why should you do it?
+
+//a. writing the code in english to understand the steps it takes to get things done properly! Helps to make a guidline.
+
+// What percentage of time should be spent thinking about how you're going to solve a problem vs actually typing in code to solve it?
+
+// a. 75-90% planning and 10-25% actually code
+
+
+
+//--------------------B. STRINGS
+// 1. Create a variable called firstVariable.
+// Assign it the value of the string "Hello World"
+// Change the value of this variable to some number.
+// Store the value of firstVariable in a new variable called secondVariable
+// Change the value of secondVariable to any string.
+// What is the value of firstVariable?
+
+let firstVariable = 'Hello World';
+//reassinging variable to number
+firstVariable = 6;
+//check
+console.log(firstVariable);
+//creating new variable equal to existing variable
+let secondVariable = firstVariable;
+//check
+console.log(secondVariable);
+//change new variable to any string
+secondVariable = 'Any String';
+//check
+console.log(secondVariable);
+//value of first variable.
+console.log(firstVariable);
+
+// 2. Create a variable called yourName and set it equal to your name as a string. Then, write an expression that takes the string "Hello, my name is " and the variable yourName so that it returns a new string with them concatenated.
+
+let yourName = 'Josh Hepworth';
+console.log('Hello, my name is ' + yourName);
+
+
+// ----------------C. BOOLEANS
+
+  const a = 4;
+  const b = 53;
+  const c = 57;
+  const d = 16;
+  const e = 'Kevin';
+
+  console.log(a < b);
+  console.log(c > d);
+  console.log('Name' =='Name');
+  console.log(true != false);
+  console.log(false == false == false == false == false == false == true);
```

Comment:
this works because of randomness, but is not the right answer, desired answer was:
`false || false || ....... || true`